inject.js

```diff
@@ -18,7 +18,12 @@ var tc = {
       vine.co
       imgur.com
       teams.microsoft.com
-      `.replace(regStrip, "")
+    `.replace(regStrip, ""),
+    blacklistrc: `\
```

Comment:
nitpick: let's use a descriptive name. it's ok to be a bit more verbose if it helps clarity in the code.

blacklistRateChange or something like that.

Reply:
I'm usually the one saying this, so I didn't follow my own advice lol. I'll update it.

Reply:
This has been removed entirely with the latest version